core/src/main/res/layout/fragment_file_issue.xml

```diff
@@ -14,10 +14,66 @@
    permissions and limitations under the License.
 -->
 
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:layout_marginLeft="20dp"
+    android:layout_marginRight="20dp"
+    android:layout_marginTop="10dp"
+    android:layout_marginBottom="10dp"
     tools:context="com.amplifyframework.devmenu.DevMenuFileIssueFragment">
 
-</FrameLayout>
\ No newline at end of file
+    <EditText
+        android:id="@+id/issue_description"
+        android:layout_width="match_parent"
+        android:layout_height="fill_parent"
+        android:autofillHints="no"
+        android:hint="@string/file_issue_hint"
+        android:inputType="textMultiLine"
+        android:layout_above="@+id/env_info_switch"
+        android:textSize="16sp" />
+
+    <Switch
+        android:id="@+id/env_info_switch"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:text="@string/env_switch_text"
+        android:layout_above="@+id/device_info_switch"
+        android:textColor="?android:attr/textColorPrimaryInverse"
+        android:textOff=""
+        android:textOn=""
```

Comment:
I think this will generate lint errors. A mechanism to state "I _intentionally_ chose for these to be blank" is to have `@string/empty` with 
```xml
<resources>
    <string name="empty" />
</resources>
```